src/sdk/client.js

```diff
@@ -10,6 +10,25 @@ const request = require('./request');
  */
 const DEFAULT_URL = 'https://zeus.ci/';
 
+/**
+ * Job statuses that we accept
+ */
+const JOB_STATUSES = {
+  PENDING: 'pending',
+  PASSED: 'passed',
+  FAILED: 'failed',
+};
+
+/**
+ * Job attributes used by Zeus
+ */
+const JOB_ATTRIBUTES = ['url', 'label', 'ref', 'status', 'result'];
+
+/**
+ * Build attributes accepted by Zeus
+ */
+const BUILD_ATTRIBUTES = ['url', 'label', 'ref'];
```

Comment:
_nit_: This is pretty nice (as long as we don't need to sanitize any of the values)! Would be nice to put this in the function, though, so it's easier to spot without scrolling all the way up.